backend/wmg/data/extract.py

```diff
@@ -29,20 +28,23 @@ def get_dataset_s3_uris():
 
         dataset_ids = []
         published_dataset_non_null_assays = (
-            session.query(Dataset.table.id, Dataset.table.assay)
+            session.query(Dataset.table.id, Dataset.table.assay, Dataset.table.organism)
             .join(Dataset.table.collection)
             .filter(
                 Dataset.table.assay != "null",
                 Dataset.table.published == "TRUE",
                 Dataset.table.is_primary_data == "PRIMARY",
                 Collection.table.visibility == "PUBLIC",
                 Dataset.table.tombstone == "FALSE",
+                Dataset.table.organism != "null",
```

Comment:
do the null and empty organisms cases also need tests?

Reply:
our schema should check for that so its unlikely to make it to the db but I noticed that null cases could break the db query for assays so I figured I would include the check for organisms too.
added a test case just to be safe